### DISCUSSION TITLE
请问如何将A账号的在线收藏转移到B账号的在线收藏

### DISCUSSION THREAD

**Discussion by endlesstear** [xiaojieonly/Ehviewer_CN_SXJ#1082] [General]
您好，请问如何将A账号的在线收藏转移到B账号的在线收藏啊？A账号注册时间较早，但由于当时不知道表站里站的区别，没注意注册方式，故无法进入里站，内有我自己分好类的收藏1w+。最近新注册了B账号，已经可以进入里站了，想将A账号的在线收藏转移到B账号，有啥办法嘛？谢谢大佬，不胜赶紧，谢谢♪ (･ω･)ﾉ

**Comment by rremo888** [comment]
先把a账号的收藏移到本地收藏，再登录b账号吧本地收藏移到a账号的收藏试试？

**Comment by xiaojieonly** [comment]
https://github.com/xiaojieonly/Ehviewer_CN_SXJ/issues/1083
